src/components/Button/test.tsx

```diff
@@ -0,0 +1,25 @@
+import { screen, render, fireEvent } from "@testing-library/react";
+import Button from ".";
+
+describe("Tests Button component", () => {
```

Comment:
```suggestion
describe("Button component | Unit test", () => {
```

esse primeiro describe acho legal esse padrao. nao tem nada de errado com o seu, só uma sugestao que acho bacana e clean